content/en/docs/examples/local-testing-with-kind/index.md

```diff
@@ -0,0 +1,388 @@
+---
+title: Testing Istio Locally With Kubernetes In Docker
+description: Example of deploying Istio locally using Kubernetes In Docker for testing and evaluation
+weight: 10
+keywords:
+- kubernetes
+- kind
+- metallb
+- cert-manager
+- ingress
+test: true
+---
+
+## About
+
+This example uses Kubernetes In Docker ([KIND][kind]) to deploy Istio locally. The cluster produced is intended to be a supplement for further testing, evaluation and experimentation with Istio.
+
+## Before you begin
+
+Make sure that you have [Docker][docker] and [Kind][kindquickstart] installed and that you are able to create and use kind clusters and connect to them with [Kubectl][kubectl].
+
+## Deploying Kubernetes
+
+For this example we'll be creating a default Kubernetes cluster using [Kind][kind] with a single node:
+
+```shell
+kind create cluster --name istio-testing
+```
```

Comment:
This (and all other code blocks) needs to be changed to use `{{< text }}>`. See documentation [here](https://istio.io/about/contribute/code-blocks/).
```suggestion
{{< text bash >}}
$ kind create cluster --name istio-testing
{{< /text >}}
```

Reply:
Updated all these spots, thank you :)